ignite/pkg/goanalysis/goanalysis.go

```diff
@@ -279,3 +279,59 @@ func createUnderscoreImport(imp string) *ast.ImportSpec {
 		},
 	}
 }
+
+// HasStructFieldsInPkg finds the struct into a package folder and checks
+// if the field exists inside the struct.
+func HasStructFieldsInPkg(pkgPath, structName string, fields []string) (bool, error) {
```

Comment:
Maybe the function should use "any" in the name because it would be true if any of the fields is present:

```suggestion
// HasAnyStructFieldsInPkg finds the struct within a package folder and checks
// if any of the fields are defined in the struct.
func HasAnyStructFieldsInPkg(pkgPath, structName string, fields []string) (bool, error) {
```
Initially I thought all of the fields must be present for the function to be true.